microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

```diff
@@ -30,7 +30,7 @@ For a webpage, you can use the [prefers-color-scheme][MDNPrefersColorScheme] CSS
     :::image-end:::  
     
     > [!IMPORTANT]
-    > Simply typing `dark` or `light` does not reveal the right tool, since there is also a way to [choose a theme for DevTools][DevtoolsCustomizeDarkTheme].  If you are wondering what to choose, make sure that you are choosing a **rendering** menu item, not an **appearance** menu item.  
+    > Simply typing `dark` or `light` does not reveal the right tool, since there is also a way to [choose a theme for DevTools][DevtoolsCustomizeTheme].  If you are wondering what to choose, make sure that you are choosing a **rendering** menu item, not an **appearance** menu item.  
 
```

Comment:
Shrink the .png below "Type emulate color, choose either Emulate CSS prefers-color-scheme: dark or Emulate CSS prefers-color-scheme: light and then select Enter."   eg:
1.  In VS Code, in this .md file, Ctrl+click the image path.  
2. Shift+Alt+R (or some menu item) to open Explorer with the image dir.  
3. Right-click the image, select Edit, opens in Paint.  
4. Select Resize, %, often 50% works, this might need 60%.  Save in Paint.
Unconfirmed whether ::: image syntax has % size options.